### PR TITLE
BLEN-510: Update readme for BlenderUSDHydraAddon

### PR DESCRIPTION
**Pull request by DagerD**
### PURPOSE
Update readme.

### TECHNICAL STEPS
Updated readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,196 +1,128 @@
...
+> git submodule update --init -f --recursive
+```
+
+### Build
+Require `python 3.10+` to be set by default.
+
+Download Blender precompiled libraries.
+
+#### Windows:
+Use Open x64 Native Tools Command Prompt for Visual Studio 2019 Community and run.
```
Comment (bnagirniak) [inline README.md:76]: Any command prompt can be used

File: README.md [hunk 2]
```diff
@@ -1,196 +1,128 @@
...
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python build.py -all -bl-libs-dir <libs_dir> -bin-dir bin -addon
```
Comment (bnagirniak) [inline README.md:81]: `-addon` isn't needed with `-all` flag

File: README.md [hunk 3]
```diff
@@ -1,196 +1,128 @@
...
+```
+
+#### Linux:
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python tools/build.py -all -bl-libs-dir <libs_dir> -bin-dir bin -addon
```
Comment (bnagirniak) [inline README.md:89]: same here: remove `-addon`

File: README.md (hunk 2 above)
Comment (bnagirniak) [reply README.md:81]: Mention where user can download <libs_dir> without loading blender git. It is possible by
`svn checkout https://svn.blender.org/svnroot/bf-blender/trunk/lib/win64_vc15` for Windows

**Review by bnagirniak — CHANGES_REQUESTED** [review]

**Review by bnagirniak — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,196 +1,134 @@
...
-```python
-from .utils import logging
-logging.limit_log('default', logging.DEBUG) # set debug level
-
-from . import config
-config.engine_use_preview = False # turn off preview of materials
-```
+# Blender USD Hydra Addon
+
+With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines and interchange, this addon is meant to add first class support for USD Hydra rendering system to Blender. This allows rendering in Blender via the USD Hydra Framework.
```
Comment (bsavery) [inline README.md:3]: Mention that its using Radeon ProRender for rendering.

**Review by VascoPi — APPROVED** [review]
Looks good.

File: README.md [hunk 5]
```diff
@@ -1,196 +1,134 @@
...
+
+With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines and interchange. This addon uses AMD Radeon™ ProRender renderer in USD Hydra rendering system in Blender.
+
+## Additional Documentation
+- [Pixar USD](https://graphics.pixar.com/usd/docs/index.html)
+- [Hydra](https://graphics.pixar.com/usd/docs/USD-Glossary.html#USDGlossary-Hydra)
+- [MaterialX](http://www.materialx.org/)
+
+## Requirements
+Currently, this addon works only with [Blender 4.0+](https://www.blender.org/download/) in Windows and Linux.
```
Comment (bsavery) [inline README.md:11]: Why can't we use Mac?

**Review by bsavery — COMMENTED** [review]
MacOS?

File: README.md (hunk 5 above)
Comment (DagerD) [reply README.md:11]: MacOS is not supported yet, we have a task for it.

**Comment by VascoPi** [comment]
Current status. Generally, HdRp build works on MacOS, but the build process requires improvement by fixing dependency issues and should be built entirely against Blender libs.

**Pull request merged**